ethportal-peertest/src/constants.rs

```diff
@@ -1,3 +1,46 @@
+use ethportal_api::{HistoryContentKey, HistoryContentValue};
+use serde_json::json;
+
+/// History HeaderWithProof content key & value
+/// Block #1000010
+pub fn test_item() -> (HistoryContentKey, HistoryContentValue) {
```

Comment:
As we deal with more than one network, I would like to see some distinction between the tests and the networks. For example, we can rename this file to `history_tests` or something or extract the test methods in a separate file.

Reply:
I agree. But, this feels a bit out of scope for this pr. There does seem to be some important refactoring to be done inside peertest. Some tests can be shared across networks, while others are network specific. IMO, a big question we need to resolve before refactoring is do we / don't we support running different groupings of subnetworks? (eg history&state&beacon or history&beacon or history, etc). This has subtle but important ramifications for a lot of our architecture and now that we're starting to expand into multiple subnetworks I think we should settle on a decision.